apps/microsoft-teams/app-actions/src/actions/send-test.ts

```diff
@@ -1,16 +1,88 @@
 import { AppActionCallContext } from '@contentful/node-apps-toolkit';
 import { AppActionCallResponse } from '../types';
+import { fetchTenantId } from '../utils';
+import helpers from '../helpers';
 
 interface AppActionCallParameters {
-  channel: string;
+  channelId: string;
+  teamId: string;
+  contentTypeId: string;
+  spaceName: string;
+}
+
+export interface BotServiceResponse {
+  ok: boolean;
+  data?: string;
+  error?: string;
 }
 
 export const handler = async (
   _payload: AppActionCallParameters,
   _context: AppActionCallContext
-): Promise<AppActionCallResponse<null>> => {
+): Promise<AppActionCallResponse<string>> => {
+  const { channelId, teamId, contentTypeId, spaceName } = _payload;
+  const {
+    cma,
+    appActionCallContext: { appInstallationId },
+  } = _context;
+
+  const config = {
```

Comment:
Let's add a todo to break this into its own file (ie something like config.ts) since we'll want to share it across actions later.